unit_test/data_driven_software_tests/test_tests.py

```diff
@@ -0,0 +1,327 @@
+
+        from neuronunit.tests.passive import TimeConstantTest as T
+        score = self.run_test(T)
+        #self.assertTrue(-1.45 < score < -1.35)
+
+
+class TestsWaveformTestCase(TestsTestCase, unittest.TestCase):
+    """Test passive validation tests"""
+
+    def test_0rheobase_parallel(self):
+        import os
+        os.system('ipcluster start -n 8 --profile=default & sleep 55 ')
+
+        #from neuronunit.optimization import data_transport_container
+
+        from neuronunit.tests.fi import RheobaseTestP as T
+        score = self.run_test(T)
+        self.prediction = score.prediction
+        self.assertTrue( score.prediction['value'] == 106.4453125 or score.prediction['value'] ==131.34765625)
+
+    def update_amplitude(test):
+        rheobase = self.prediction['value']#first find a value for rheobase
+        test.params['injected_square_current']['amplitude'] = rheobase*1.01
+
+    def test_missing(self):
+        print(dir(self.model))
+        print(self.model.get_AP_thresholds())
+        print(self.model.get_AP_amplitudes())
+        print(self.model.get_AP_widths())
+
+    def test_5ap_width(self):
+        #from neuronunit.optimization import data_transport_container
+
+        from neuronunit.tests.waveform import InjectedCurrentAPWidthTest as T
+        self.update_amplitude(T)
+        score = self.run_test(T)
+        #self.assertTrue(-0.6 < score < -0.5)
+
+    def test_6ap_amplitude(self):
+        #from neuronunit.optimization import data_transport_container
+        from neuronunit.tests.waveform import InjectedCurrentAPAmplitudeTest as T
+        self.update_amplitude(T)
+
+        score = self.run_test(T)
+        #self.assertTrue(-1.7 < score < -1.6)
+
+    def test_7ap_threshold(self):
+        #from neuronunit.optimization import data_transport_container
+
+        from neuronunit.tests.waveform import InjectedCurrentAPThresholdTest as T
+        self.update_amplitude(T)
+
+        score = self.run_test(T)
+        #self.assertTrue(2.25 < score < 2.35)
+
+
+class TestsFITestCase(TestsTestCase, unittest.TestCase):
+    """Test F/I validation tests"""
+
+    #@unittest.skip("This test takes a long time")
+    # def test_rheobase_serial(self):
+        #from neuronunit.optimization import data_transport_container
+
+    #    from neuronunit.tests.fi import RheobaseTest as T
+    #    score = self.run_test(T)
+        #self.assertTrue(0.2 < score < 0.3)
+
+    #@unittest.skip("This test takes a long time")
+
+    def test_0rheobase_parallel(self):
+        import os
+        os.system('ipcluster start -n 8 --profile=default & sleep 35 ')
+
+        #from neuronunit.optimization import data_transport_container
+
+        from neuronunit.tests.fi import RheobaseTestP as T
+        score = self.run_test(T)
+        #score.prediction
+        self.prediction = score.prediction
+        self.assertTrue( score.prediction['value'] == 106.4453125 or score.prediction['value'] ==131.34765625)
+
+
```

Comment:
@russelljjarvis Everything below this line appears to be a copy of existing tests.  Please remove them.